src/test/fuzz/tx_pool.cpp

```diff
@@ -142,7 +144,7 @@ FUZZ_TARGET_INIT(tx_pool_standard, initialize_tx_pool)
         return c.out.nValue;
     };
 
-    while (fuzzed_data_provider.ConsumeBool()) {
+    FUZZER_CONTROLLED_LOOP(fuzzed_data_provider) {
```

Comment:
it would be nice to be able to pass the limit to the loop (in the same line). Otherwise it will get messy if there are nested loops and one forgets to adjust the limit or the loops are moved without moving the limit, which is in a different line.

Reply:
Sure -- replaced the extra #define variable with a new macro argument

Reply:
Maybe we should just drop the "default 1000" one and always explicitly set the maximum number of iterations?

Reply:
Yeah, I wouldn't mind that. An additional `, 1000` doesn't seem too verbose.